compositor_pipeline/src/pipeline/output.rs

```diff
@@ -160,32 +173,36 @@ impl OutputOptionsExt<RawDataReceiver> for RawDataOutputOptions {
 impl Output {
     pub fn frame_sender(&self) -> Option<&Sender<PipelineEvent<Frame>>> {
         match &self {
-            Output::Rtp { encoder, .. } => encoder.frame_sender(),
-            Output::EncodedData { encoder } => encoder.frame_sender(),
+            Output::Rtp { encoder, .. }
```

Comment:
personal opinion(here and below), but I think is less readable than the previous version and  very annoying when you need to change anything